includes/connection/class-wc-terms.php

```diff
@@ -49,11 +49,11 @@ public static function register_connections() {
 										'toType'        => $tax_object->graphql_single_name,
 										'fromFieldName' => $tax_object->graphql_plural_name,
 										'resolve'       => function( $source, array $args, AppContext $context, ResolveInfo $info ) use ( $tax_object ) {
-											$resolver = new TermObjectConnectionResolver( $source, $args, $context, $info, $tax_object->name );											
-                                            
+											$resolver = new TermObjectConnectionResolver( $source, $args, $context, $info, $tax_object->name );
+
 											// Get the term ids that are associated with this $source
 											$terms = wp_list_pluck( get_the_terms( $source->ID, $tax_object->name ), 'term_id' );
-											
```

Comment:
There was repeated whitespace here that my IDE deleted. Decided to not re-add them back in.